src/main/res/values/strings.xml

```diff
@@ -169,10 +169,35 @@
 
     <string name="intro_title_6">Ready</string>
     <string name="intro_desc_6">You\'re all set to go. Thanks for using Amahi.</string>
-
-    <string name="intro_title_4">Play your music</string>
+    <string name="intro_title_4">Play Your Music</string>
     <string name="title_selected_server">Server</string>
-
+    <string name="connection">CONNECTION</string>
+    <string name="shares">shares</string>
+    <string name="unknown_artist">Unknown Artist</string>
+    <string name="android_amahi_anywhere">Android Amahi Anywhere</string>
+    <string name="image_mediatype">image/*</string>
+    <string name="md5">MD5</string>
+    <string name="prepare_dialog_2">prepare_dialog</string>
+    <string name="progressdialog">progress_dialog</string>
+    <string name="prepare_dialog_3">prepare_dialog</string>
```

Comment:
Is this and others (identical!) strings that the users see?

What should it be @octacode?

Reply:
They are not seen by the user. Putting them in strings.xml means they would be translated if we choose to localize the Amahi app in the future.